worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/FileWriter.java

```diff
@@ -154,19 +155,27 @@ public void decrementPendingWrites() {
 
   @GuardedBy("flushLock")
   protected void flush(boolean finalFlush) throws IOException {
+    MemCacheManager cacheManager = MemCacheManager.getMemCacheManager();
```

Comment:
We can make `cacheManager` a member of `FileWriter`, and initialize in the constructor of `FileWriter`, we can get `CelebornConf` there.